web/src/compose_popovers.js

```diff
@@ -68,6 +69,11 @@ export function initialize() {
     popover_menus.register_popover_menu(".compose_control_menu_wrapper", {
         placement: "top",
         onShow(instance) {
+            const compose_text_length = compose_validate.check_overflow_text();
+            let disable_buttons = false;
```

Comment:
This variable name is too general. Rename it something like `disable_on_hover` for more clarity. We can maybe do better since it's not obvious which buttons this is for, but it least this makes it clear the buttons are disabled on hover only

Reply:
Renamed to `disable_on_hover`, I propose renaming it to `disable_poll_todo_buttons_on_hover` for a clearer and more descriptive name.

Reply:
A variable simply named `empty_compose` should be best, I now realise. Both clear and versatile enough for any different uses in the future.

Also, this change should be in the commit moving over the poll button to a place where it can go into the popover, not the second, as that way the 1st commit would be buggy.